task/sast-snyk-check/0.1/sast-snyk-check.yaml

```diff
@@ -41,7 +41,9 @@ spec:
           value: $(params.ARGS)
       script: |
         #!/usr/bin/env bash
+        set -euo pipefail
         . /utils.sh
+        trap 'handle_error $(results.TEST_OUTPUT.path)' ERR
         SNYK_TOKEN="$(cat /etc/secrets/snyk_token)"
```

Comment:
This will fail with -e, we should check emptyness of file instead?

Reply:
https://github.com/redhat-appstudio/build-definitions/pull/587/commits/4281c2f1d2f0f93ccdc7f0530bfebbff5d8ea99f I addressed it here